### PR TITLE
Fixed : Playback speed reset on forwarding video

### PR DESCRIPTION
**Pull request by Kronos-2701**
I have fixed the issue #757 .
Kindly review and merge the code.

### CODE REVIEW

File: lib/src/cupertino/cupertino_controls.dart [hunk 1]
```diff
@@ -754,6 +757,10 @@ class _CupertinoControlsState extends State<CupertinoControls>
     final skip =
         (_latestValue.position - const Duration(seconds: 15)).inMilliseconds;
     controller.seekTo(Duration(milliseconds: math.max(skip, beginning)));
+    log("Reversing the video", name: "Video Player");
```
Comment (diegotori) [inline lib/src/cupertino/cupertino_controls.dart:760]: Please remove all logging. Thanks.

File: lib/src/cupertino/cupertino_controls.dart [hunk 2]
```diff
@@ -754,6 +757,10 @@ class _CupertinoControlsState extends State<CupertinoControls>
...
+    Future.delayed(const Duration(milliseconds: 200), () {
```
Comment (diegotori) [inline lib/src/cupertino/cupertino_controls.dart:761]: You may want to explain why we're delaying setting the playback speed by 200 ms.

**Review by diegotori — CHANGES_REQUESTED** [review]
Also, if this is happening on Material, please address it there as well.

Thanks.

File: lib/src/cupertino/cupertino_controls.dart (hunk 2 above)
Comment (Kronos-2701) [reply lib/src/cupertino/cupertino_controls.dart:761]: Yes, I added it to ensure a smooth transition .

File: lib/src/cupertino/cupertino_controls.dart (hunk 1 above)
Comment (Kronos-2701) [reply lib/src/cupertino/cupertino_controls.dart]: All the logging has been removed now. 
Thanks for the feedback.

File: lib/src/cupertino/cupertino_controls.dart [hunk 3]
```diff
@@ -754,6 +756,9 @@ class _CupertinoControlsState extends State<CupertinoControls>
     final skip =
         (_latestValue.position - const Duration(seconds: 15)).inMilliseconds;
     controller.seekTo(Duration(milliseconds: math.max(skip, beginning)));
+    Future.delayed(const Duration(milliseconds: 1000), () {
```
Comment (diegotori) [inline lib/src/cupertino/cupertino_controls.dart:759]: Add comments above this line explaining why there needs to be a one second delay before setting the playback speed.

That way, any future dev beyond yourself will be better able to understand why its being done this way.

File: lib/src/cupertino/cupertino_controls.dart [hunk 4]
```diff
@@ -762,6 +767,9 @@ class _CupertinoControlsState extends State<CupertinoControls>
     final skip =
         (_latestValue.position + const Duration(seconds: 15)).inMilliseconds;
     controller.seekTo(Duration(milliseconds: math.min(skip, end)));
+    Future.delayed(const Duration(milliseconds: 1000), () {
```
Comment (diegotori) [inline lib/src/cupertino/cupertino_controls.dart:770]: Same here. Add comments above this line explaining why there needs to be a one second delay before setting the playback speed.

That way, any future dev beyond yourself will be better able to understand why its being done this way.

File: lib/src/cupertino/cupertino_controls.dart (hunk 3 above)
Comment (Kronos-2701) [reply lib/src/cupertino/cupertino_controls.dart:759]: Hi Diego , 
I've added the added explaining the reason for adding a delay of 1 second in the code.
Kindly review it.
Thanks and Regards

File: lib/src/cupertino/cupertino_controls.dart (hunk 4 above)
Comment (Kronos-2701) [reply lib/src/cupertino/cupertino_controls.dart:770]: Added comments.

**Comment by Kronos-2701** [comment]
Hi Diego, 
Sorry for the delay. I've made the changes required.

**Review by diegotori — APPROVED** [review]

**Comment by diegotori** [comment]
LGTM. Once the checks are green, then I'll evaluate them either tonight or tomorrow night.

**Comment by Kronos-2701** [comment]
Hey @diegotori  , 
All Checks are green. Could you please check and merge.
Thanks and regards

**Comment by diegotori** [comment]
@Kronos-2701 sorry for the delay. I'll get to them tomorrow.

**Pull request merged**